modules/overrides/overrides.go

```diff
@@ -71,6 +87,14 @@ func NewOverrides(defaults Limits) (*Overrides, error) {
 	var manager *runtimeconfig.Manager
 	subservices := []services.Service(nil)
 
+	metricDefaultsLimits.WithLabelValues("max_local_traces_per_user").Set(float64(defaults.MaxLocalTracesPerUser))
```

Comment:
make `max_local_traces_per_user`, etc. constants

Reply:
This is addressed in the last commit.

Reply:
I'm still seeing individual static strings.

Reply:
Oh sorry, I thought you meant `prometheus.MustNewConstMetric` to make them constants.  You mean make the metric name string a constant to be referenced by both the defaults and the overrides, right?

Reply:
yup. I just meant to make a constant string that they both use. I have no idea what `MustNewConstMetric` is but let's just follow whatever Cortex does here.

Reply:
Yep that's what they are doing.  This `promehteus.Collector` pattern is new to me, but the const bit is document here: https://prometheus.io/docs/instrumenting/writing_exporters/#collectors

Reply:
In Cortex the exporter is also a module, but as it is now we aren't doing that.  If we want to do that, I think we just need to make a `limits/exporter` package and glue it up, but not sure how far to go with modules.  It would be our first "exporter" module, but it could be a pattern if we want to structure the exporters as modules.  Seems to me that they belong with the module they support, rather than a module in themselves.  What do you think?